### DISCUSSION TITLE
Game lags any time I get a notification

### DISCUSSION THREAD

**Discussion by SinumOfficial** [ppy/osu#17454] [Q&A]
### Type

Performance

### Bug description

I have the brand new Galaxy S22 Ultra with 1TB of storage, and basically when I'm playing the game runs fine until I get a notification from anything and it'll start lagging for no apparent reason.

### Version

2022.319.0-lazer

### Logs

n/a

**Comment by frenzibyte** [comment]
Note that mobile is not a current focus, desktop always comes first. However, the best you can do is attaching a video with the frame graph open (Ctrl-F11 twice, via external keyboard) and replicate the issue. Right now this is not actionable and as such been converted to a discussion thread.

**Comment by peppy** [comment]
This sound like a device specific issue. It is likely deprioritising the game to render the android UI, or something in the background on your phone is consuming resources after the notification arrives.

**Reply by frenzibyte** [reply]
Ah, I was under the thought this is referring to in-game notifications, not OS-level notifications.

**Reply by SinumOfficial** [reply]
Well, I just got this phone, and it had the update that fixes the bottlenecking, like.. I got the phone yesterday so I'm not sure what to say. 😂 I even had the game mode set to performance and everything, not sure at all.
